Challenges/python/dominant.py

```diff
@@ -0,0 +1,96 @@
+#!/bin/python3
+
+import math
+import os
+import random
+import re
+import sys
+
+
+
+#
+# Complete the 'numCells' function below.
+#
+# The function is expected to return an INTEGER.
+# The function accepts 2D_INTEGER_ARRAY grid as parameter.
+#
+
+def isDominantCell(grid, xPos, yPos):
+    thisValue = grid[xPos][yPos]
+    neighbors = getNeighborValues(grid, xPos, yPos)
+
+    for val in neighbors:
+        if val >= thisValue:
+            return False
+    
+    return True
+
+def getNeighborValues(grid, xPos, yPos):
+    neighborValueList = []
+    
+    # for xOffset in range(-1, 1):
```

Comment:
Didn't wanna waste time so just went for the hacky approach after the loop version didn't want to work for whatever reason